server/conf/evolutions/default/24.sql

```diff
@@ -0,0 +1,4 @@
+
+# --- !Ups
+
+ALTER TYPE block_synchronization_state ADD VALUE 'STORING_REWARDS';
```

Comment:
This can go in the previous evolution, it makes no sense to add two evolutions in a single commit, unless they are huge and you want to split them logically.